packages/pds/src/index.ts

```diff
@@ -175,6 +177,23 @@ export class PDS {
 
     const labelCache = new LabelCache(db)
 
+    let contentReporter: ContentReporter | undefined = undefined
+    if (config.unacceptableHandleWordsB64) {
+      contentReporter = new ContentReporter({
+        backgroundQueue,
+        moderationService: new ModerationService(
```

Comment:
I don't love how this ended up (instantiating a service here) - but I was having difficulty finding the correct ordering of these 🤔 

I guess one option would be to move the content reporting functionality to the moderation service...

Reply:
That would def work!  This isn't so bad either—personally not too worried about it either way!